packages/shared/src/vault/__tests__/__snapshots__/executeSwap.test.ts.snap

```diff
@@ -19,7 +19,7 @@ exports[`executeSwap can be invoked with localnet options 1`] = `
 [
   [
     3,
-    "0x74623171676539767664326d6d6a7866687578757132303468346678787068723076666e736e78323035",
+    "0x626372743170376d6e6c6c36717570346c336c6367677668377432366d3467727961776a793064306367616e7a683665347a6a6d326433727471743975737178",
```

Comment:
snapshot changed bc i needed to pass a different btc address (we previously passed a testnet address, but it needs to be a regtest on backspin)